src/tools/immutable_object_cache/ObjectCacheStore.h

```diff
@@ -47,13 +49,20 @@ class ObjectCacheStore {
   int handle_promote_callback(int, bufferlist*, std::string);
   int do_evict(std::string cache_file);
 
+  TokenBucketThrottle* create_throttle();
+  void apply_qos();
+  void take_token_from_throttle(uint32_t token_num);
+  void handle_throttle_ready(int r, C_SaferCond* cond, uint64_t f);
+
   CephContext *m_cct;
   RadosRef m_rados;
   std::map<uint64_t, librados::IoCtx> m_ioctx_map;
   ceph::mutex m_ioctx_map_lock =
     ceph::make_mutex("ceph::cache::ObjectCacheStore::m_ioctx_map_lock");
   Policy* m_policy;
   std::string m_cache_root_dir;
+  bool m_qos_enabled_flag{false};
```

Comment:
Nit: technically unnecessary if you just use the existence of the throttle as proof that it's enabled.

Reply:
done. thanks